build.assets/Dockerfile-fips

```diff
@@ -78,5 +81,46 @@ RUN mkdir -p /opt && cd /opt && curl -L https://github.com/gravitational/libbpf/
 COPY pam/ /opt/pam_teleport/
 RUN make -C /opt/pam_teleport install
 
+ARG LLVM_VERSION=7.0.1
```

Comment:
As @zmb3 mentioned, please add a comment to explain that we need this exact LLVM version, not to update it by mistake.